File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        
        int cnt =0; 
        vector<int> arr(26,0);

        for(auto e:s) {
            arr[e-'a']++ ; } 

        for(auto e:t){
            arr[e-'a']-- ; } 

        for(auto e:arr){ 
            cnt += abs(e) ; } 

        return cnt ; 

        
        
    }
};

File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        


        cnt =0 
        arr= [0]*27
        for e in s:
            arr[ord(e)- ord('a') ] +=1 

        for e in t: 
            arr[ord(e) - ord('a') ] -=1 

        return sum( abs(e)   for e in arr )  